public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,183 @@
+
+
+```d
+shared static this() {
+  // ...
+  auto settings = new HTTPServerSettings;
+  // Provide a default port in case of the $PORT variable isn't set.  
+  settings.port = environment.get("PORT", "8080").to!ushort;
+  listenHTTP(settings, router);
+}
+```
+
+You also need to create a Procfile, which is a text file in the root directory of your application, responsible to explicitly declare what command 
+should be executed to start your app.
+
+The Procfile in the example app looks like this:
+
+```
+web: ./hello-world
+```
+
+### 2 : Prepare the app 
+
+Before going further you should login to the heroku cli.
+
+To login you have to use the [heroku toolbelt](https://toolbelt.heroku.com/standalone).
+
+This provides you access to the Heroku Command Line Interface (CLI), which can be used for managing and scaling your applications and add-ons.
+
+After installing the toolbet just run 
+
+```
+$ heroku login
+```
+
+### 3 : Create the app 
+
+To do so you can go on the [heroku dashboard](https://dashboard.heroku.com) and create a new app. 
+After doing so memorize the name of your app, you will use it later. 
+
+
+
+or use the command line like this 
+
+```
+$ heroku create
+Creating app... done, ⬢ rocky-hamlet-67506
+https://rocky-hamlet-67506.herokuapp.com/ | https://git.heroku.com/rocky-hamlet-67506.git
+```
+
+The app's name here is rocky-hamlet-67506. 
+
+### Deploy using git 
+
+To deploy your app you will use git and push your code to heroku as if you were pushing on your git repo. 
+
+You need to add the git remote with the name of your app. 
+
+As shown in the previous section, our name's app is rocky-hamlet-67506. But change it to yours. 
+
+```
+$ heroku git:remote -a rocky-hamlet-67506
+```
+
+We can see now that the remote is added to our git config
+
+```
+$ git remote -v
+heroku	https://git.heroku.com/rocky-hamlet-67506.git (fetch)
+heroku	https://git.heroku.com/rocky-hamlet-67506.git (push)
+```
+
+### Adding the buildpack
+
+Buildpacks are responsible for transforming deployed code into a slug, 
+which can then be executed on a dyno. Buildpacks are composed of a set of scripts, 
+and depending on the programming language, the scripts will retrieve dependencies, 
+output generated assets or compiled code, and more.
+
+[More information](https://devcenter.heroku.com/articles/buildpacks)
+
+We are going to use this [webpack](https://github.com/skirino/heroku-buildpack-vibe.d) it uses dmd by default. 
+
+```
+$ heroku buildpacks:set https://github.com/skirino/heroku-buildpack-vibe.d.git#cedar-14
+```
+
+You should also create a file called `vibed_buildpack.config` in the root of your app. 
+We are doing this because this buildpack's dependencies are outdated. 
+
+Fortunately we can override them by providing new urls : 
+
+```
+DMD_ARCHIVE_URL = http://downloads.dlang.org/releases/2016/dmd.2.070.2.zip
+DUB_ARCHIVE_URL = http://code.dlang.org/files/dub-0.9.25-linux-x86_64.tar.gz
+```
+
+### Deploy the code 
+
```

Comment:
no text. Maybe sth. like:

You can proceed in your usual git habit and write awesome code. Once you want to release a  new version, you just push the newest version to the `heroku` endpoint.